app/src/main/res/layout/browser_overlay.xml

```diff
@@ -2,78 +2,90 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<org.mozilla.focus.browser.BrowserNavigationOverlayScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/browser_overlay_background"
-    android:orientation="vertical"
-    android:descendantFocusability="afterDescendants"
-    android:paddingTop="27dp"
-    >
+    android:fillViewport="true"
+    android:focusable="false"
+    android:scrollbars="none">
+    <!-- Fill viewport? When don't have enough tiles. -->
 
     <LinearLayout
```

Comment:
Nit: comment that scrollview can only have one child? Maybe this isn't necessary, but looked a little inefficient.

Reply:
I think it can have multiple children: it's designed this way because ScrollView is a FrameLayout and we wanted a LinearLayout.